From 09cbe3cf8fdf6a9dc6d9789f4d2416c619e8bc89 Mon Sep 17 00:00:00 2001
From: KrIr17 <elendil.krir17@gmail.com>
Date: Mon, 6 Jul 2026 13:55:08 +0200
Subject: [PATCH] Fix incorrect Poppler version check

1ad89c941bad5e42a3378cd8644a4c4cd3b6ea71 adapts to changes in Poppler
26.06.0, so that should what be the version we check for.

Identified by @PBS in https://gitlab.com/inkscape/inkscape/-/merge_requests/7968#note_3526749691
---
 src/extension/internal/pdfinput/pdf-input.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/extension/internal/pdfinput/pdf-input.cpp b/src/extension/internal/pdfinput/pdf-input.cpp
index 0d46a0577f..3bfa55dd86 100644
--- a/src/extension/internal/pdfinput/pdf-input.cpp
+++ b/src/extension/internal/pdfinput/pdf-input.cpp
@@ -846,7 +846,7 @@ PdfInput::add_builder_page(std::shared_ptr<PDFDoc>pdf_doc, SvgBuilder *builder,
     }
 
     // Apply crop settings
-#if POPPLER_CHECK_VERSION(26, 2, 0)
+#if POPPLER_CHECK_VERSION(26, 6, 0)
     std::optional<PDFRectangle> clipToBox;
 #else
     _POPPLER_CONST PDFRectangle *clipToBox = nullptr;
@@ -865,7 +865,7 @@ PdfInput::add_builder_page(std::shared_ptr<PDFDoc>pdf_doc, SvgBuilder *builder,
     }
 
     std::optional<PDFRectangle> cropBox;
-#if POPPLER_CHECK_VERSION(26, 2, 0)
+#if POPPLER_CHECK_VERSION(26, 6, 0)
     cropBox = clipToBox;
 #else
     if (clipToBox) {
-- 
GitLab

